### PR TITLE
sidemenu is changed

### PR DESCRIPTION
**Pull request by vaqif14**

### CODE REVIEW

File: sidemenu.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SideMenu</title>
+</head>
+<body>
+    <h1>SideMenu</h1>
```
Comment (vaqif14) [inline sidemenu.html]: bunu sil

Comment (vaqif14) [reply sidemenu.html]: @Alimdar213  bunu sil

Comment (Alimdar213) [reply sidemenu.html]: Yaaxsi

**Review by vaqif14 — COMMENTED** [review]
ok

**Pull request merged**